engine/src/main/java/org/terasology/reflection/copy/CopyStrategyLibrary.java

```diff
@@ -148,6 +159,15 @@ public void clear() {
     }
 
     /**
+     * @return a copy of the this library that uses the specified stategy for the specified type.
+     */
+    public <T>  CopyStrategyLibrary createCopyOfLibraryWithStrategy(Class<T> type, CopyStrategy<T> strategy) {
```

Comment:
It is only used to create a single copy. It is not used to add all the copy strategies